src/components/SigninForm/SigninForm.jsx

```diff
@@ -0,0 +1,198 @@
+import React from 'react'
+import {
+    Box,
+    Typography,
+    Button,
+    TextField,
+    InputAdornment,
+    Checkbox,
+    Link,
+    styled,
+} from '@mui/material'
+import backgroundImage from 'Images/heroSen.jpg'
+import PersonIcon from '@mui/icons-material/Person'
+import LockRoundedIcon from '@mui/icons-material/LockRounded'
+import RadioButtonUncheckedIcon from '@mui/icons-material/RadioButtonUnchecked'
+import CheckCircleOutlineIcon from '@mui/icons-material/CheckCircleOutline'
+
+const InputText = styled(TextField)(() => ({
+    outline: '0.1rem solid #69AD28',
+    borderRadius: '0.5rem',
+}))
```

Comment:
You need write this into another file example: SigninForm.styled.js, and then import to use it